e2e/next/src/next.test.ts

```diff
@@ -35,8 +36,8 @@ describe('Next.js Applications', () => {
     runCLI(`generate @nrwl/react:lib ${reactLib} --no-interactive`);
     runCLI(`generate @nrwl/js:lib ${jsLib} --no-interactive`);
 
-    // Create file in public that should be copied to dist
-    updateFile(`apps/${appName}/public/a/b.txt`, `Hello World!`);
+    // Create file in public that should not be copied to dist since it's not declared
+    updateFile(`apps/${appName}/a/b.txt`, `Hello World!`);
```

Comment:
Can you leave this one in `public`? We copy the `public` folder from source to dist, and then the assets should be in addition to that.